### PR TITLE
 Added mongoose App

### PR DESCRIPTION
**Pull request by Rohitrky2021**

### CODE REVIEW

File: express-mongoose_Rohit_Yadav/ts.js
Comment (Sonichigo) [inline express-mongoose_Rohit_Yadav/ts.js:1]: Is there any need of this empty file ?

File: express-mongoose_Rohit_Yadav/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+npm install
+```
+
+**2. Run the Development Server:**
+Run the following command to launch the development server:
+
+```Bash
+npm run dev
+```
+
```
Comment (Sonichigo) [inline express-mongoose_Rohit_Yadav/README.md:24]: can you also add the steps needed, after running npm run dev and how to run with keploy ?

File: express-mongoose_Rohit_Yadav/keploy-logs.txt
Comment (Sonichigo) [inline express-mongoose_Rohit_Yadav/keploy-logs.txt:1]: this is an empty file. you can remove this.

**Review by Sonichigo — CHANGES_REQUESTED** [review]

File: express-mongoose_Rohit_Yadav/README.md (hunk 1 above)
Comment (Rohitrky2021) [reply express-mongoose_Rohit_Yadav/README.md:149]: I was actually having a problem setting up WSL in my Windows, that's why. Now I have updated it properly.

**Review by Sonichigo — APPROVED** [review]
LGTM 🚀

**Pull request merged**